spec/controllers/queue_items_controller_spec.rb

```diff
@@ -159,4 +159,128 @@
     end
   end
 
+  describe 'PUT :update' do
+
+    let(:user) { user = Fabricate(:user) }
+    let(:queue_items) { queue_items = user.queue_items }
+    let(:first) { first = queue_items.first }
+    let(:sec) { sec = queue_items[1] }
+    let(:third) { third = queue_items.last }
+    before do
+      3.times do |i|
+        Fabricate(:queue_item, user: user, order_value: i+1)
+      end
+    end
+
+    context 'for authorized user' do
+      before do
+        login_user user
+      end
+      context 'for valid order values' do
+        it 'updates the order of the queue items' do
+          post :update_queue, queue_items: [  {id: first.id, order_value: 1},
+                                        {id: sec.id, order_value: 3},
```

Comment:
This and next spec should have these additional queue_items params aligned to the first - like you have on lines 198 - 201